docs/maui/views/GravatarImageSource.md

```diff
@@ -0,0 +1,260 @@
+---
+title: GravatarImageSource - .NET MAUI Community Toolkit
+author: GeorgeLeithead
+description: "`GravatarImageSource` allows you to use as an Image source, a users Gravatar registered image via their email address."
+ms.date: 08/05/2022
+---
+
+# GravatarImageSource
+
+A _Gravatar_ (a "globally recognized avatar") is an image that can be used on multiple websites as your avatar — that is, an image that represents you. For example, a Gravatar can identify a person in a forum post, in a blog comment, and so on. (You can register your own Gravatar at the Gravatar website at [http://www.gravatar.com/](http://www.gravatar.com/).) If you want to display images next to people's names or email addresses, you can use GravatarImageSource.
+
+## Syntax
+The following example shows how to use `GravatarImageSource`:
+
+```xaml
+<ContentPage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.MyPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit">
+    <VerticalStackLayout>
+        <Image>
+            <Image.Source>
+                <toolkit:GravatarImageSource
+                    CacheValidity="1"
+                    CachingEnabled="True"
+                    Email="youremail@here"
+                    Image="MysteryPerson" />
+            </Image.Source>
+        </Image>
+    </VerticalStackLayout>
+</ContentPage>
+```
+
+The equivalent C# code is:
+
+```csharp
+using CommunityToolkit.Maui.Views;
+
+partial class MyPage : ContentPage
+{
+	public MyPage()
+	{
+        Image myImage = new()
+        {
+            Source = new GravatarImageSource()
+            {
+                CacheValidity = TimeSpan.FromDays(1),
+                CachingEnabled = true,
+                Email = "youremail@here",
+                Image= DefaultImage.MysteryPerson
+            },
+        };
+		Content = myImage;
+	}
+}
+```
+
+## Properties
+
+| Property | Type | Description |
+|---|---|---|
+| CacheValidity | `TimeSpan` | The `CacheValidity` property, of type `TimeSpan`, specifies how long the image will be stored locally for. The default value of this property is 1 day. |
+| CachingEnabled | `bool` | The `CachingEnabled` property, of type `bool`, defines whether image caching is enabled. The default value of this property is `true`. |
+| Email | `string?` | The `Email` property, of type `string?`, specifies the gravatar account email address.  If unset, the Gravatar image is rendered. If set and not found on Gravatar, the `Image` property image will be rendered. |
+| Image | `DefaultImage` | The `Image` property, of type `DefaultImage` is an enumeration that is used to specify the default image if the `email` is not found on Gravatar. |
```

Comment:
Perhaps we can link to the example below which shows the possible values for the `Image` property?

Reply:
Added link to example.